src/debug_menu.cpp

```diff
@@ -60,6 +60,11 @@
 #include "rng.h"
 #include "signal.h"
 
+#if defined(TILES)
+#include "cata_tiles.h"
+extern std::unique_ptr<cata_tiles> tilecontext;
```

Comment:
Instead of including `cata_tiles` here, I'd suggest adding `game::save_screenshot()`. It could be noop in curses, or even save a textual "screenshot" in the future.